docs/decisions/0001-adoption-of-an-off-the-shelf-code-of-conduct.md

```diff
@@ -0,0 +1,34 @@
+---
+nav_order: 1
+parent: Decision Records
+
+status: accepted
+date: 2023-01-04
```

Comment:
Which date this one is @timgentry ?

Reply:
That was the date we adopted the code of conduct. As I mentioned, this comes from a historical, long-lived branch that needs merging into main in sensible chunks. Hope that makes sense? Thanks